docs/DatabaseCatalogAPI.yaml

```diff
@@ -78,7 +80,10 @@ paths:
       parameters:
         - $ref: "#/components/parameters/limitQueryParam"
         - $ref: "#/components/parameters/offset"
-        - $ref: "#/components/parameters/filter"
+        - $ref: "#/components/parameters/provider"
+        - $ref: "#/components/parameters/country_code"
+        - $ref: "#/components/parameters/subdivision_name"
+        - $ref: "#/components/parameters/municipality"
         - $ref: "#/components/parameters/sort"
```

Comment:
There's also filtering by producer_URL according to https://github.com/MobilityData/mobility-feed-api/issues/33
It could be delegated to a new issue.

Reply:
added to all feeds endpoints